testfiles/SingleFamilyHouse_HP_Slab.idf

```diff
@@ -0,0 +1,5453 @@
+!-Generator IDFEditor 1.51
```

Comment:
This file hasn't been added to the `testfiles/CMakeLists.txt` file, so this has never been run by our automated tests.  It will be interesting to see if it has errors/warnings once they are added.

OK, I went ahead and added them to the CMakeLists file, and ran them.  The error files have lots of messages, including a severe error, that need to be cleaned up before 9.5 is released.  @lymereJ please open an issue to track this and any other issues that need to be addressed before 9.5.